src/gtk/slider.cpp

```diff
@@ -312,32 +312,24 @@ bool wxSlider::Create(wxWindow *parent,
         return false;
     }
 
-    const bool isVertical = (style & wxSL_VERTICAL) != 0;
+    const bool isVertical = (style & (wxSL_LEFT | wxSL_RIGHT | wxSL_VERTICAL)) != 0;
```

Comment:
This should be ideally renamed to something else (`hasHorzLabels`?) as the name is misleading now and it's not totally clear what/why are we testing here.